packages/ui/src/components/modals/ActionCallbackResultDropdown.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react'
```

Comment:
Could you please add a comment to this file describing what it's used for, and from which page/context ?

I think these types of comments would make the UI code a lot easier to navigate for people not as familar with the UI.

Ditto for other new tsx files in the PR.

Reply:
I can describe what it's used for but not what uses it. The latter is breaking encapsulation and bound to get out of date. Someone can easily import the file in another location and would not know to update the comment and it would be misleading.

Similar to my complaints about Lars' comments. Adding comments doesn't affect how you navigate code it only adds extra information to that code. It may better inform you but not change how the editors link and navigate files.  If you're using "find" for a certain comment name you think is next to a piece of code, instead use "find" for the actual code.